src/lib/registration/helpers.rb

```diff
@@ -85,11 +85,18 @@ def self.service_description(service)
       (descr && !descr.empty?) ? "#{descr} (#{url})" : url
     end
 
+    # run the network configuration module
     def self.run_network_configuration
       log.info "Running network configuration..."
       Yast::WFM.call("inst_lan", [{ "skip_detection" => true }])
     end
 
+    # check if the network configuration module is present
+    # @return [Boolean] true if the network configuration client is present
+    def self.network_configurable
+      Yast::WFM.ClientExists("inst_lan")
```

Comment:
It depends, by default it's installed. The check makes sure the client (and thus the `yast2-network`) package is installed. In installation we are pretty sure it's present, but in installed system user might remove it.

This check just guards the `Yast::WFM.call("inst_lan")` call to avoid possible errors.